manage-participants/process_queue.yml

```diff
@@ -41,6 +41,12 @@
         name: "../../requirements_roles/infra-ansible/roles/scm/git"
       vars:
         action: push
+        git_config:
+          name: Git Bot
+          username: git-bot
+          email: git-bot@no-reply
+          message: "Remove processed jobs from queue"
```

Comment:
would be great if this could include some more specific info about which one was processed

Reply:
@oybed, I can add a counter?

Adding names here would be messy as it's a single commit for all jobs processed

Reply:
Was hoping that the job or file name from the queue was easily available to append to this message. Don't think we want to "compose" this, but if already available in a fact, please append it.